.github/workflows/ci.yaml

```diff
@@ -662,6 +601,11 @@ jobs:
           # - persistence-static-report (WIP)
     steps:
       - uses: actions/checkout@master
+
+      - uses: actions/setup-node@v4
+        with:
```

Comment:
Why here "with" instead of "node-version-file" used above?

Reply:
hä? they are all:

I moved the node install up to the start of the action run here to make it consistent with the other jobs in the pipeline.

```yaml
      - uses: actions/setup-node@v4
        with:
          node-version-file: '.nvmrc'
```

Reply:
Ah, I see: In the others there are some more lines removed, so overlooked the "with".